cernopendata_client/cli.py

```diff
@@ -152,7 +158,25 @@ def get_file_locations(server, recid, doi, title, protocol, expand):
     type=click.STRING,
     help="Which CERN Open Data server to query? [default=http://opendata.cern.ch]",
 )
-def download_files(server, recid, doi, title, protocol, expand):
+@click.option(
+    "--filter-name",
+    "name",
+    type=click.STRING,
+    help="Download file that matching the name",
```

Comment:
"Download files matching exactly the file name"

(there can be several, we'll see it in the future when we shall be having subdirectories...)